dev-packages/e2e-tests/test-applications/vue-3/tests/performance.test.ts

```diff
@@ -122,3 +122,101 @@ test('sends a pageload transaction with a route name as transaction name if avai
     },
   });
 });
+
+test('sends a lifecycle span for the tracked HomeView component - with `<>`', async ({ page }) => {
```

Comment:
This is a bit of an unfortunate and very intransparent limitation of our e2e test setup but we shouldn't navigate to the same route in more than one e2e test. The reason is that we execute them in parallel in CI and the `waitForTransaction` call from one test might actually resolve for a transaction in another test, depending on the condition in the callback. 

To avoid this confusion, we generally create a new route for each test. Suggestion: Either we create new routes for the two new tests or we integrate them in already existing tests and simply check for the spans there. I'll let you make the call, I'm fine with either option. 
Also, feel free to test both kinds `trackComponent` notations in one route/test :)

Reply:
hey, thanks for the info. i decided to go with creating a new route, and grouped the different notations together in a single test.